geppetto-ui/src/components/plot/showcase/PlotShowcase.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import model from './model.json';
+import PlotComponent from "../PlotComponent";
+import ExternalInstance from '@geppettoengine/geppetto-client/js/geppettoModel/model/ExternalInstance';
+
+export default class PlotShowcase extends Component {
+    constructor (props) {
+        super(props);
+        this.instancePath = "nwbfile.acquisition.test_sine_1";
+        GEPPETTO.Manager.loadModel(model);
+        Instances.getInstance(this.instancePath);
+        Instances.getInstance(`${this.instancePath}.data`);
+        Instances.getInstance(`${this.instancePath}.timestamps`);
+    }
+
+    getLegendName (projectId, experimentId, instance, sameProject) {
+        const instancePath = instance.getInstancePath()
+            .split('.')
+            .filter((word, index, arr) => index != 0 && index != arr.length - 1)
+            .join('.');
+
+        if (sameProject) {
+            window.Project.getExperiments().forEach(experiment => {
```

Comment:
Why using experiments to get legend name?

Reply:
We may just use the instance name, or set up a lookup table if we want to be sophisticated